### DISCUSSION TITLE
photon and Docker tile server

### DISCUSSION THREAD

**Discussion by edwh** [komoot/photon#826] [Q&A] [answered]
I'm using the OSM tile server Docker container from https://github.com/Overv/openstreetmap-tile-server

Does this under the covers have a Nominatim database which I can therefore configure Photon to automatically update from?

Apologies for some ignorance of the ecosystem.

**Comment by mtmail** [comment, marked as answer]
It doesn't. Tile servers and geocoding servers cannot really share a database, they import data into different table structures. So you'd need to set up Nominatim separately.

**Comment by edwh** [comment]
Very clear, thank you.

**Comment by simonpoole** [comment]
Actually you can run a nominatim database in the same database _installation_ (docker or otherwise) as a tile server, you just need to rename one of the two (SQL) databases. But you, naturally, don't share any of the data in that scenario, so it doesn't really make a lot of sense with docker.
